libraries/AP_HAL_ChibiOS/hwdef/JHEMCU-GSF405A-RX2/hwdef-bl.dat

```diff
@@ -0,0 +1,3 @@
+# JHEMCU GSF405A target for external receiver
```

Comment:
You will need to build a bootloader for this as well and add it to the PR

Reply:
Done